config/plugins/webhooks/gtn/script.js

```diff
@@ -142,6 +145,25 @@
                         }
                         removeOverlay();
                     });
+
+                    var gtn_workflows = $("#gtn-embed").contents().find("span[data-workflow]");
+                    // Buttonify
+                    gtn_workflows.addClass("galaxy-proxy-active");
+
+                    gtn_workflows.click((e) => {
+                        var target = e.target;
+
+                        // Sometimes we get the i or the strong, not the parent.
+                        if (e.target.tagName.toLowerCase() !== "span") {
+                            target = e.target.parentElement;
+                        }
+
+                        trs_url = target.dataset.workflow;
```

Comment:
yes, the intent was it would be marked up like:

```
<span class="workflow" data-workflow="https://training.galaxyproject.org/training-material/api/ga4gh/trs/v2/tools/metagenomics-mothur-miseq-sop-short/versions/workflow1_quality_control"><strong>Quality Control</strong> <i class="fas fa-share-alt" aria-hidden="true"></i></span>
```

There's an example live here: https://training.galaxyproject.org/training-material/topics/metagenomics/tutorials/mothur-miseq-sop-short/tutorial.html#hands-on-importing-and-launching-a-workflow

Reply:
Yep, I tried one of the random pages and that's where I saw the `data-workflow` and not `data-trs-url` and made the swap.  Cool, just making sure that was the standard to expect, this works now.

Reply:
Ah, perfect, yeah, guess I changed that at a later time and forgot to update this PR. Appreciate the second set of eyes since I'm not working at capacity lately.